### PR TITLE
Update UG for command history

### PR DESCRIPTION
**Pull request by rexcyrio**

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -254,32 +251,34 @@ Examples:
...
 
 Format: `find KEYWORD [MORE_KEYWORDS]`
 
-- The search is case-insensitive. e.g `hans` will match `Hans`
-- The order of the keywords does not matter. e.g. `Hans Bo` will match `Bo Hans`
-- Only the name is searched.
-- Only full words will be matched e.g. `Han` will not match `Hans`
+- The search is case-insensitive. e.g., `alpha` will match `Alpha`
+- The order of the keywords does not matter. e.g. `Ris Pasir` will match `Pasir Ris`
+- All person fields are search.
```
Comment (minosx31) [inline docs/UserGuide.md:263]: Should it be `searched` (past tense) instead?

File: docs/UserGuide.md [hunk 2]
```diff
@@ -326,6 +326,24 @@ Undoes the last modification done to AIMS.
...
+  you can navigate back to it by pressing the down arrow (`↓`) multiple times.
+- If you're already at the *earliest* command, pressing the up arrow (`↑`) will move the cursor to the *beginning* of
+  the text in the command box instead.
+- If you're already at the *latest* command, pressing the down arrow (`↓`) will move the cursor to the *end* of the text
+  in the command box instead.
+- At any point in the command history, you can simply edit the command and re-run it. Note that once you navigate away
+  from this point in the command history, the changes you've made to this current command in history will be lost.
+- The command history file is located at `[JAR file location]/data/command_history.txt`. Whenever you run a command *
+  *successfully**, the command history file will be automatically updated. In other words, commands that fail to run due
+  to syntax errors or otherwise will**not** be recorded in the command history.
```
Comment (minosx31) [inline docs/UserGuide.md:345]: I think there is an accidental line break at `successfully` so it doesn't appear bolded.

And also there should be a space between before `**not**`

**Review by minosx31 — CHANGES_REQUESTED** [review]
Just a few typos but the rest looks good!

File: docs/UserGuide.md (hunk 2 above)
Comment (rexcyrio) [reply docs/UserGuide.md]: Good catch! For the `**successfully**` issue, I think its because IntelliJ auto inserts line breaks into the markdown file 😢

**Review by minosx31 — APPROVED** [review]

**Comment by rexcyrio** [comment]
:fire:

**Pull request merged**